webrtc.html

```diff
@@ -2848,12 +2817,24 @@
                     </ol>
                   </li>
                   <li>
-                    <p>Set <var>connection</var>'s <a>[[\IsClosed]]</a> slot to
-                    <code>true</code>.</p>
+                    <p>Set the <code><a data-link-for=
+                    "RTCDtlsTransport">state</a></code> of each of
```

Comment:
[state](https://w3c.github.io/webrtc-pc/#dom-rtcdtlstransport-state) is a read-only attribute and cannot be set.